src/library-extender/README.md

```diff
@@ -0,0 +1,92 @@
+# Custom Digital Assets Management provider
+
+> IMPORTANT - Please note that there is a known limitation, you cannot use the Angular binding syntax {{item.data.CreatedBy}}, when creating component HTML templates, you must use another binding, for example [textContent]="item.data.CreatedBy", or [innerHtml]="item.data.Content".
+
+You can add custom DAM providers if the one you are using is not supported by default from Sitefinity - Frontify and Cloudinary.
```

Comment:
Lets not specify the default providers. We may add new ones in the future and this will become invalid. We may put a link to the SF docs if such

Reply:
Default providers are removed